### PR TITLE
Experimental Web Authentication support in STP 71

### PR DESCRIPTION
**Pull request by daytonlowell**
Source: https://webkit.org/blog/8517/release-notes-for-safari-technology-preview-71/

### CODE REVIEW

File: features-json/webauthn.json [hunk 1]
```diff
@@ -196,7 +196,7 @@
       "11":"n",
       "11.1":"n",
       "12":"n",
-      "TP":"p d #2"
+      "TP":"a d #2"
```
Comment (HM100) [inline features-json/webauthn.json]: ditto

**Comment by daytonlowell** [comment]
What's the difference between `d` and `n`? I did `a` because it appears to only support a specific class of devices for this feature.

File: features-json/webauthn.json [hunk 2]
```diff
@@ -327,7 +327,8 @@
   },
   "notes":"",
   "notes_by_num":{
-    "1":"Can be enabled at `about:flags`\r\n\r\nEdge 13 used an earlier draft syntax. As of Edge 14 the implementation is prefixed and based on the FIDO 2.0 Web APIs."
+    "1":"Can be enabled at `about:flags`\r\n\r\nEdge 13 used an earlier draft syntax. As of Edge 14 the implementation is prefixed and based on the FIDO 2.0 Web APIs.",
+    "2":"Can be enabled using the Develop > Experimental Features menu. Currently supprots USB-based CTAP2 devices."
```
Comment (Fyrd) [inline features-json/webauthn.json]: Typo fix:
```suggestion
    "2":"Can be enabled using the Develop > Experimental Features menu. Currently supports USB-based CTAP2 devices."
```

**Comment by Fyrd** [comment]
Have updated manually.

**Pull request closed without merging**